app/server/app-configurations.ts

```diff
@@ -8,7 +8,11 @@ import helmet from 'helmet';
 import config from 'config';
 import { tyaNunjucks } from './controllers/content';
 import { dateFormat } from './utils/dateUtils';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
 import { ContentSecurityPolicyOptions } from 'helmet/dist/types/middlewares/content-security-policy';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
should these commented out lines be removed?

Reply:
I didn't change these files i think this was from when i rebased off master and nightly dev - i can remove it though